packages/bpk-component-card/src/BpkCardWrapper.module.scss

```diff
@@ -58,4 +59,11 @@
       transform: rotate(-45deg);
     }
   }
+
+  &:has(.bpk-card-wrapper--header:active) {
```

Comment:
It needs to be `.bpk-card-wrapper:has(.bpk-card-wrapper--header:active)::after` because of the structure of the component's elements:
```
<div class='bpk-card-wrapper'>
  <div class='bpk-card-wrapper--header'></div>
  <div class='bpk-card-wrapper--content'></div>
  ::after
</div>
```
The box shadow that we want to control is added on the `::after` pseudo-element so that's where we want to add the box shadow when the header is active. For these reason the current solution is:
- When the `bpk-card-wrapper` has a header element that is active, then add the box shadow to the `::after` element (which comes after the header and content elements).
- If we apply the box shadow only to `.bpk-card-wrapper--header:active` then it will add the box shadow to the header element which we don't want. If we add the box shadow to `bpk-card-wrapper:active::after` then it will add the shadow any time any part of the card is clicked which we also don't want because we only want to add it when the header is active.